.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "✨Feature request"
+about: Suggest an idea for this project
```

Comment:
Suggest a feature?

Reply:
Agreed, `idea` seems a bit out of place.